src/styles/components/classroom-manager.styl

```diff
@@ -0,0 +1,20 @@
+.classroom-manager
+  &__table
+    background-color: white
+    border: solid 2px #DEDEDE
+
+  .table
+    &__caption
+      bottom: -2.25em
+      font-weight: bold
+      position: relative
+      text-align: left
+
+    &__row-header
+      background-color: GREY_5
+      border-top: solid 2px #DEDEDE !important
```

Comment:
Minor observation: I assume the `!important` here is to overcome Grommet's bizarrely specific `.grommetux-background-color-index--light .grommetux-table th, .grommetux-background-color-index--dark .grommetux-background-color-index--light .grommetux-table th { border-color: ... }`. (Three .classes strong in specificity, really?) While I'm mildly uncomfortable with `!important` in the CSS, I can understand the necessity when working with a UX framework.

Minor question: How come #DEDEDE doesn't get called GREY_4? 🤷‍♂️